src/services/goToDefinition.ts

```diff
@@ -78,6 +78,10 @@ namespace ts.GoToDefinition {
             });
         }
 
+        if (isPropertyName(node) && hasEffectiveModifier(node.parent, ModifierFlags.Override)) {

```

Comment:
Sorry. I'm not 100% sure. But the original issue said:

>triggered on the override keyword

I guess it's means 

```ts

class C extends B {
  over/*here*/ride foo () {}
}
```

/cc: @uniqueiniquity

Reply:
@Kingwl Yes, that's correct. I did not intend for the method name to have this behavior.